zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -1410,10 +1426,9 @@ impl<'a, F: Field> ConstraintBuilder<'a, F> {
             Some(condition) => lookup.conditional(condition.clone()),
             None => lookup,
         };
-
         let compressed_expr = self.split_expression(
             "Lookup compression",
-            rlc::expr(&lookup.input_exprs(), self.power_of_randomness),
+            rlc::expr(&lookup.input_exprs(), self.lookup_powers_of_randomness),
             MAX_DEGREE - IMPLICIT_DEGREE,
         );
         self.store_expression(name, compressed_expr, CellType::Lookup(lookup.table()));
```

Comment:
This seems need to be stored in `LookupPhase3` since it contains challenge after phase 2